src/api/api.cc

```diff
@@ -11065,27 +11065,31 @@ static const char* gRecordReplayKnownFeatures[] = {
 
   // Record/replay events are turned off by default (for now) (RUN-1251)
   "disable-collect-events"
-};
+});
+
+// The set of all experimental flags pertaining to features we are currently developing.
+// Ideally, this should always be a short list.
+// NOTE: These should generally be "double-negative" flags which we need to convert to positive in the near future.
+static std::vector<std::string>* gExperimentalFlags = new std::vector<std::string>({
+  "disable-collect-events"
+});
```

Comment:
Can this be a static array instead?  That seems simpler than trying to define a pointer to a vector.